### PR TITLE
fix: (liquidity): Missing user readable error messages

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/views/AddLiquidity/index.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useCallback, useEffect, useState } from 'react'
 import { BigNumber } from '@ethersproject/bignumber'
 import { TransactionResponse } from '@ethersproject/providers'
-import { Currency, currencyEquals, ETHER, TokenAmount, WETH } from '@pancakeswap/sdk'
+import { Currency, currencyEquals, ETHER, TokenAmount, WETH, JSBI } from '@pancakeswap/sdk'
```
Comment (chefilip) [inline src/views/AddLiquidity/index.tsx:4]: Could you help me fix eslint on this PR? Thanks

Comment (memoyil) [reply src/views/AddLiquidity/index.tsx]: Done

**Comment by chefilip** [comment]
Screen Shot 2022-07-04 at 13 03 39

Good PR @memoyil  !

Could you help me update the border color of Prices and pool share box as the same as the border color above? 

Thank bro

**Comment by memoyil** [comment]
> Screen Shot 2022-07-04 at 13 03 39
> 
> Good PR @memoyil !
> 
> Could you help me update the border color of Prices and pool share box as the same as the border color above?
> 
> Thank bro

Done 👍

**Review by chefilip — APPROVED** [review]

**Comment by memoyil** [comment]
> Screen Shot 2022-07-04 at 13 03 39
> 
> Good PR @memoyil !
> 
> Could you help me update the border color of Prices and pool share box as the same as the border color above?
> 
> Thank bro

Separate that fix to this pr https://github.com/pancakeswap/pancake-frontend/pull/4199

File: src/components/Card/index.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const Card = styled(Box)<{
 export default Card
 
 export const LightCard = styled(Card)`
-  border: 1px solid ${({ theme }) => theme.colors.cardBorder};
+  border: 1px solid ${({ theme }) => theme.colors.background};
```
Comment (chef-jojo) [inline src/components/Card/index.tsx:19]: Override back the update

Comment (memoyil) [reply src/components/Card/index.tsx]: Done

**Comment by chef-jojo** [comment]
Lgtm
Let's schedule this release next week

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**